### PR TITLE
Lab3

### PR DESCRIPTION
**Pull request by Reqvel**

### CODE REVIEW

File: static/css/progress-bar.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+}
+  
+.progress-bar:focus {
+    outline: none;
+}
+  
+/*webkit*/
+.progress-bar::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    width: 13.53px;
```
Comment (katetushkan) [inline static/css/progress-bar.css]: noooo, pls noooo, Don't use float numbers for px, browser engine can count it wrong

File: static/css/progress-bar.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    border-radius: 0px 0 0 0px;
+    margin: -0px 0 -0px -0px;
+    background: #FFFFFF;
+    border: 0;
+    border-right-width: 0;
+} 
+
+@media only screen and (max-width: 1070px) {
+    .progress-bar-volume {
+        width: 9vw;
```
Comment (katetushkan) [inline static/css/progress-bar.css]: use rem, vw can be tricky on mobile phones, especially in Safari for now better use % and in the future new units will be available for correct usage of viewport.

File: static/css/music-player.css [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+  .footer {
+    height: 70px;
+  }
+
+  .progress-time-played {
+    display: none;
+  }
+
+  .progress-time-total {
+    width: 3vw;
```
Comment (katetushkan) [inline static/css/music-player.css]: same here, for vw, check at the end of the review, pls

File: static/css/base.css [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+        rgba(56, 56, 56, 0.83) 17.32%, 
+        #1C1C1C 100%);
+}
+
+html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, p, 
+blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, 
+img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, 
+i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, 
+caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, 
+embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby,
```
Comment (katetushkan) [inline static/css/base.css]: use universal selector for it - `*`

**Review by katetushkan — CHANGES_REQUESTED** [review]
Do like what u did, pls fix things that I mentioned and i'l give u 9. If u wanna upgrade add focus styles to ur buttons and links. Let me know what u wanna do.

**Comment by Reqvel** [comment]
Did fix the issues.  Don't really know what to add on focus, so 9 is fine

**Review by katetushkan — APPROVED** [review]
`9` is urs.

**Pull request merged**